docs/releases/4.1.txt

```diff
@@ -219,6 +219,9 @@ Models
   for :ref:`persistent database connections <persistent-database-connections>`
   in order to reduce the number of failed requests, e.g. after database server
   restart.
+* New :meth:`BaseConstraint.validate(instance, exclude=None, using=DEFAULT_DB_ALIAS) <.BaseConstraint.validate>`
+  method returns whether the instance validate the constraint or not. This needs
+  to be implemented for each subclasses.
```

Comment:
I _think_ `validate` here ought to be `validates` or `validated`?
`each subclasses` should be `each subclass` or `all subclasses` I believe.

Reply:
Agreed, we also still need to determine if we actually want to force all subclasses to implement this form of validation.